Src/RE2.Managed/Regex2.cs

```diff
@@ -219,7 +220,11 @@ public static void ClearRegexes()
         ///
         /// </code>
         /// </example>
-        public static unsafe bool Matches(string pattern, string text, out List<Dictionary<string, string>> matches)
+        public static unsafe bool Matches(
+            string pattern,
+            string text,
+            out List<Dictionary<string, string>> matches,
+            out List<Dictionary<string, FlexMatch>> matchIndices)
```

Comment:
you dont need two structures if im not mistaken.
you could use flexmatch as well. flexmatch contains the property to hold the value.

Reply:
Yes, I will remove the redundant structure.

Reply:
Done.